ubi_manifest/worker/tasks/content_audit.py

```diff
@@ -113,13 +113,13 @@ def content_audit_task() -> None:
                         out_repo.ubi_config_version,
                     )
                     output_blacklist.extend(parse_blacklist_config(config))
-                    whitelist, debuginfo_whitelist = filter_whitelist(
-                        config, output_blacklist
-                    )
-                    output_whitelist |= whitelist | debuginfo_whitelist
-                    output_whitelist |= {
-                        f"{md.name}:{md.stream}" for md in config.modules.whitelist
-                    }
+                    pkg_whitelist, _ = filter_whitelist(config, output_blacklist)
+                    output_whitelist |= pkg_whitelist
+                    # don't include module whitelist for source or debug repos
+                    if (out_repo.id.count("debug") + out_repo.id.count("source")) == 0:
```

Comment:
I guess it would better just to skip getting the md/mdd and following comparison for debug and source repos. Instead of this little hack.

Reply:
Even if fetching md/mdd content is skipped `config.modules.whitelist` persists and missing modules will be reported in log, right? So we still need to skip the addition of modules.whitelist into the greater whitelist.
But it could save time to skip those md/mdd content searches as well.

Reply:
I've added a condition to enable skipping module content code.